docs/self-managed/platform-deployment/openshift-helm.md

```diff
@@ -0,0 +1,194 @@
+
+Any version not explicitly marked in the table above is not tested, and we cannot guarantee compatibility.
+
+## Quick start
+
+You can find a quick start guide in the [Helm chart repository itself](https://github.com/camunda/camunda-platform-helm/tree/main/openshift).
+
+## Limitations
+
+The `Elasticsearch`, `Keycloak`, and `Postgresql` charts all specify default non-root users for security purposes. In order to deploy these charts through the `camunda-platform` chart, these default values must be removed. Unfortunately, due to a [longstanding bug in Helm](https://github.com/helm/helm/issues/9136) that affects all Helm versions from 3.2.0 and greater, this makes the installation of the chart (when deploying any of these sub-charts) more complex. Note that this is only an issue if you are deploying `Elasticsearch`, `Keycloak` (via `Identity`), or `Postgresql` (via `Keycloak`). If you are not deploying these, or not via the `camunda-platform` chart, or you are using a [permissive SCC](./openshift.md#permissive-scc).
+
+:::note
+This also affects installations done through the OpenShift console, as it still uses Helm under the hood.
+:::
+
+## Configuration
+
+As discussed in the [OpenShift deployment guide](./openshift.md), you will need to configure the pod and container security contexts based on your desired security context constraint (aka SCC).
+
+### Permissive SCC
+
+If you wish to use a permissive SCC, then you can install the charts as they are. Simply follow the [general Helm deployment guide](./kubernetes-helm.md).
+
+### Restrictive SCC
+
+If you wish to use more a restrictive SCC, then you will need to configure the following minimum set of values for the various applications. The recommendations outlined in the general [OpenShift deployment guide](./openshift.md) hold here as well. As the Camunda 8 applications do not define a pod or security context, follow these recommendations, or simply omit defining any.
+
+If you are deploying with a SCC where `RunAsUser` is `MustRunAsRange` (e.g. the default `restricted` SCC), and are deploying at least one of `Elasticsearch`, `Keycloak`, or `Postgresql`, then we will need to unset the default security context of these charts. If this does not apply to you, then you can stop here.
+
+How we will do this depends on which Helm version you use: 3.1.3 and lower, or 3.2.0 and greater (i.e. one affected by [this bug](https://github.com/helm/helm/issues/9136)). You can find out your Helm version by running the following:
+
+```shell
+$ helm version
+version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.17.5"}
+```
+
+#### Helm 3.1.3 or lower
+
+If you're running on Helm 3.0.0 up to 3.1.3, then you will need to add these values to your `values.yaml` file, or save them to a new file locally, e.g. `openshift.yaml`, also available [here](https://github.com/camunda/camunda-platform-helm/blob/main/openshift/values.yaml):
```

Comment:
For better usability and accessibility, it's [recommended](https://webaccess.berkeley.edu/ask-pecan/click-here) to avoid using `here` for hyperlinks and put the link under a descriptive name.

e.g.
> also a predefined [values file](https://github.com/camunda/camunda-platform-helm/blob/main/openshift/values.yaml) is available
> also you can use this predefined [values.yaml](https://github.com/camunda/camunda-platform-helm/blob/main/openshift/values.yaml) file